### ISSUE TITLE
[Bug]: ioHub cannot find tracker

### ISSUE THREAD

**Issue by Nicobruno92** [psychopy/psychopy#6196]
### PsychoPy Version

2023.2.3

### What OS are your PsychoPy running on?

Windows 11

### Bug Description

This does not work. I have been trying to make it work with the current distribution of psychopy (2023.2) and nothing makes sense been trying to fix it without success,  and not being able to narrow it down. 

My basic code:
```
`from psychopy import core, visual
from psychopy.iohub import launchHubServer

# Configure the eyetracker devicepip
iohub_config = {
    "eyetracker.hw.gazepoint.gp3.EyeTracker": {
        "name": "tracker",
        "enable": True,
        "save_events": True,
        "stream_events": True,
        "event_buffer_length": 1024,
        "monitor_event_types": ['BinocularEyeSampleEvent', 'FixationStartEvent', 'FixationEndEvent'],
        "calibration": {
            "target_duration": 1.25,
            "target_delay": 0.5
        },
        "model_name": "GP3",
        "manufacturer_name": "GazePoint"
    }
}

# Start the ioHub server with the configuration
io = launchHubServer(**iohub_config)
tracker = io.devices.tracker

# Setup a window for calibration
win = visual.Window(fullscr=True)

# Start the calibration process
tracker.runSetupProcedure(win)

# Main experiment code goes here

# Cleanup
win.close()
io.quit()
`
```

This is the error:

> *** iohub warning: Display / Monitor unit type has not been set.
> Error during device creation ....
> Traceback (most recent call last):
>   File "C:\Users\cocud\anaconda3\envs\experimental\lib\site-packages\psychopy\iohub\server.py", line 715, in createNewMonitoredDevice
>     dev_data = self.addDeviceToMonitor(dev_cls_name, dev_conf)
>   File "C:\Users\cocud\anaconda3\envs\experimental\lib\site-packages\psychopy\iohub\server.py", line 849, in addDeviceToMonitor
>     _dconf = yload(open(dev_conf_pth, 'r'), Loader=yLoader)
> FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\cocud\\anaconda3\\envs\\experimental\\lib\\site-packages\\psychopy\\iohub\\devices\\eyetracker\\hw\\gazepoint\\gp3\\default_eyetracker.yaml'
> Error during device creation ....
> Traceback (most recent call last):
>   File "C:\Users\cocud\anaconda3\envs\experimental\lib\site-packages\psychopy\iohub\server.py", line 715, in createNewMonitoredDevice
>     dev_data = self.addDeviceToMonitor(dev_cls_name, dev_conf)
>   File "C:\Users\cocud\anaconda3\envs\experimental\lib\site-packages\psychopy\iohub\server.py", line 849, in addDeviceToMonitor
>     _dconf = yload(open(dev_conf_pth, 'r'), Loader=yLoader)
> FileNotFoundError: [Errno 2] No such file or directory: 'C:\\Users\\cocud\\anaconda3\\envs\\experimental\\lib\\site-packages\\psychopy\\iohub\\devices\\eyetracker\\hw\\gazepoint\\gp3\\default_eyetracker.yaml
> During handling of the above exception, another exception occurred:
> 
> Traceback (most recent call last):
>   File "C:\Users\cocud\anaconda3\envs\experimental\lib\site-packages\psychopy\iohub\server.py", line 649, in _addDevices
>     self.createNewMonitoredDevice(dev_cls_name, dev_conf)
>   File "C:\Users\cocud\anaconda3\envs\experimental\lib\site-packages\psychopy\iohub\server.py", line 728, in createNewMonitoredDevice
>     raise ioHubError('Error during device creation ....')
> psychopy.iohub.errors.ioHubError: ioHubError:
> Args: ('Error during device creation ....',)
> 
> _addDeviceView: Error adding class. 
> Traceback (most recent call last):
>   File "C:\Users\cocud\anaconda3\envs\experimental\lib\site-packages\psychopy\iohub\client\__init__.py", line 1081, in _addDeviceView
>     dev_import_result = import_device(dev_mod_pth, dev_cls_name)
>   File "C:\Users\cocud\anaconda3\envs\experimental\lib\site-packages\psychopy\iohub\devices\__init__.py", line 933, in import_device
>     device_class = getattr(module, device_class_name)
> AttributeError: module 'psychopy.iohub.devices.eyetracker.hw.gazepoint.gp3' has no attribute 'EyeTracker'
> Traceback (most recent call last):
>   File "c:\Users\cocud\Documents\wandering-mind\SART\asdd.py", line 24, in <module>
>     tracker = io.devices.tracker
> AttributeError: 'ioHubDevices' object has no attribute 'tracker'
> 

### Expected Behaviour

connected to the eye-tracker and run calibration.

### Steps to Reproduce

Run the code with the current version of psychopy and the last version of https://github.com/psychopy/psychopy-eyetracker-gazepoint

**Comment by mdcutone** [comment]
We're fixing this issue for the next release.

**Comment by HanZhang-psych** [comment]
Just following up on this. It seems that we are getting the same issue on version 2024.1.5.

**Comment by HanZhang-psych** [comment]
Hello, any new updates on this?

**Comment by peircej** [comment]
Could you check on release 2024.2.0 (or wait for 24.2.1 later this week as we know of a couple of bug fixes on the way)?
https://github.com/psychopy/psychopy/releases

**Comment by peircej** [comment]
@HanZhang-psych I'm closing this as I believe it is resolved in 2024.2.0 so please go ahead and try that. Of course, reopen this issue if it turns out the bug is still there. Thanks

**Comment by villmaria** [comment]
Hi @peircej, 
I am experiencing the same issue on a M1 MacBook Air running macOS Sequoia 15.1. 
I have followed the docs listed her verbatim here using the Pupil Labs Neon: https://docs.pupil-labs.com/neon/data-collection/psychopy/
This issue persists on Psychopy 2024.2.0 and 2024.2.4. I began using by 2024.2.4 but rolled back to 2024.2.0 as per the this tread. Do you have any suggestions on resolving this?

**Comment by peircej** [comment]
@villmaria I believe this was fixed from 2024.2.0 onwards (2024.2.4 should also be fine). Could you provide the full error traceback please and also the version of the pupil-labs plugin you have installed? Also are you running on standalone psychopy or your own installation?
